advanced/helm/pattern-1/apim-with-analytics/templates/azure/azure-rbac.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if (.Values.cloudProvider) and eq .Values.cloudProvider "Azure" }}
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
License header year should be 2019. Change in all relevant, new files.

Reply:
Fixed